collect_app/src/androidTest/java/org/odk/collect/android/regression/FillBlankFormWithRepeatGroupTest.java

```diff
@@ -0,0 +1,197 @@
+                .swipeToNextQuestion()
+                .clickSaveAndExit();
+
+        new MainMenuPage(main)
+                .startBlankForm("form5")
+                .inputText("T1")
+                .closeSoftKeyboard()
+                .swipeToNextQuestion()
+                .inputText("T2")
+                .closeSoftKeyboard()
+                .swipeToNextQuestion()
+                .inputText("T3")
+                .closeSoftKeyboard()
+                .swipeToNextQuestion()
+                .clickSaveAndExit();
+
+        new MainMenuPage(main)
+                .startBlankForm("form6")
+                .inputText("T1")
+                .closeSoftKeyboard()
+                .swipeToNextQuestion()
+                .inputText("T2")
+                .closeSoftKeyboard()
+                .swipeToNextQuestion()
+                .inputText("T3")
+                .closeSoftKeyboard()
+                .swipeToNextQuestion()
+                .clickSaveAndExit();
+
+        new MainMenuPage(main)
+                .startBlankForm("form7")
+                .swipeToNextQuestion()
+                .clickSaveAndExit();
+
+        new MainMenuPage(main)
+                .startBlankForm("form8")
+                .closeSoftKeyboard()
+                .swipeToNextQuestion()
+                .clickSaveAndExit();
+
+        new MainMenuPage(main)
+                .startBlankForm("form9")
+                .closeSoftKeyboard()
+                .swipeToNextQuestion()
+                .clickSaveAndExit();
+    }
+
+    @Test
+    public void repeatGroupNotAdded_ShouldBackButtonBeActive() {
+
+        //TestCase6
+        new MainMenuPage(main)
+                .clickOnMenu()
+                .clickGeneralSettings()
+                .clickOnUserInterface()
+                .clickNavigation()
+                .clickUseSwipesAndButtons()
+                .pressBack(new GeneralSettingsPage(main))
+                .pressBack(new MainMenuPage(main))
+                .startBlankFormWithRepeatGroup("RepeatGroupAndGroup")
+                .clickOnDoNotAddGroup(new FormEntryPage("RepeatGroupAndGroup", main))
+                .closeSoftKeyboard()
+                .clickBackwardButton()
+                .clickOnDoNotAddGroup()
+                .closeSoftKeyboard()
+                .swipeToNextQuestion()
+                .clickSaveAndExit();
+    }
+
+    @Test
+    public void firstQuestionWithLongLabel_ShouldDisplayBothAnswersInHierarchyPage() {
+
+        //TestCase11
+        new MainMenuPage(main)
+                .startBlankForm("basic")
+                .inputText("1")
+                .closeSoftKeyboard()
+                .swipeToNextQuestion()
+                .inputText("2")
+                .closeSoftKeyboard()
+                .clickOnGoToIconInForm()
+                .checkIsTextDisplayed("2")
+                .clickJumpEndButton()
+                .clickSaveAndExit();
+    }
+
+    @Test
+    public void openHierarchyPageFromLastView_ShouldNotDisplayError() {
```

Comment:
This could be something like `openingHierarchyPageFromLastView_doesNotDisplayError`.

Reply:
As I remember, we've determined that the names contain 'Should'

Reply:
Happy to leave `should` in then if that's the case. It doesn't really change the meaning of the test.